### PR TITLE
Profiles added to index page

### PR DESCRIPTION
**Pull request by samajammin**
## Description
Add fellow profiles to the index page

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -1,35 +1,39 @@
...
 import {
   PageBody,
-  Section,
+  // Section,
   ButtonLink,
+  Center,
   H2,
 } from "../components/SharedStyledComponents"
 import horzLogo from "../images/fellowship-program-logo.svg"
+import fellowsMap from "../images/fellows-map.png"
```
Comment (samajammin) [inline src/pages/index.js:15]: I'd prefer if we used Gatsby Image for these images. Gatsby Image does a bunch of image optimization out of the both which improves performance.

**Review by samajammin — COMMENTED** [review]
See comment - no need to block deploy if this is urgent but let's please refactor this to use Gatsby Image:
https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-image/

File: src/pages/index.js (hunk 1 above)
Comment (lassejaco) [reply src/pages/index.js:15]: Totally agree! I also wanted to update Gatsby to v3, and iirc there's a different way to handle optimized images there - didn't want to start us out on a deprecated path. I'll get that done asap, just wanted to get this out for review by team quickly.

Comment (samajammin) [reply src/pages/index.js:15]: Sounds good. FYI although it Gatsyb Image does change a bit, you can upgrade to v3 without changing anything re: Gatsby Image, e.g. see @wackerow's update on the ESP site: https://github.com/ethereum/esp-website/pull/84

**Pull request merged**